someSong/src/pages/question-details/question-details.ts

```diff
@@ -56,17 +56,25 @@ export class QuestionDetailsPage {
         this.question.user = user.val();
       });
 
-      for (let answer of question.val().answers)
+      if (question.answers)
       {
-        this._answer.getAnswerDetails(answer).then(ans => {
-          var answer = ans.val();
-          this._user.getUser(answer.user).then(user => {
-            answer.user = user.val();
+        for (let answer of question.answers)
+        {
+          this._answer.getAnswerDetails(answer).then(ans => {
```

Comment:
see comment in the #8 pull request, only instead of `userID`, pass a `questionID`